### PR TITLE
[MOBL-1241] Show the in-app close icon based on the value coming in the payload

### PR DESCRIPTION
**Pull request by rahulrvp**
The in-app message's close button is customizable using the payload now.

### CODE REVIEW

File: android-sdk/src/main/java/com/blueshift/util/InAppUtils.java [hunk 1]
```diff
@@ -349,6 +350,75 @@ private static JSONObject getCloseButtonJSONObject(Context context, InAppMessage
         return null;
     }
 
+    public static TextView getCloseButtonIconTextView(Context context, InAppMessage inAppMessage, int iconHeight) {
+        int defaultTextSize = 22;
+        String defaultText = "\uF00D";
+        String defaultColor = "#ffffff";
+        String defaultBgColor = "#00ffffff";
```
Comment (ketanshikharebsft) [inline android-sdk/src/main/java/com/blueshift/util/InAppUtils.java:357]: Is this hash code correct?

**Review by ketanshikharebsft — APPROVED** [review]
LGTM

File: android-sdk/src/main/java/com/blueshift/util/InAppUtils.java (hunk 1 above)
Comment (rahulrvp) [reply android-sdk/src/main/java/com/blueshift/util/InAppUtils.java:357]: Yes

Comment (rahulrvp) [reply android-sdk/src/main/java/com/blueshift/util/InAppUtils.java:357]: I will double-check it anyways before the merge

**Pull request merged**